tensorboard/components/tf_storage/storage.ts

```diff
@@ -112,8 +113,8 @@ export function makeBindings<T>(
   getObserver: (key: string, options: AutoStorageOptions<T>) => Function;
   disposeBinding: () => void;
 } {
-  const hashListeners = [];
-  const storageListeners = [];
+  const hashListeners: ListenKey[] = [];
+  const storageListeners: ListenKey[] = [];
   function get(key: string, options: StorageOptions<T> = {}): T {
     const {defaultValue, useLocalStorage = false} = options;
     const value = useLocalStorage
```

Comment:
For line 123, please change it to :

```
return value == undefined ? _.cloneDeep(defaultValue)! : fromString(value)!;
```

Reply:
So this PR wasn't up-to-dated when you're reviewing it. I created the internal cl but didn't update here. Do you prefer me to update here or create internal cl? cl/469760530

Reply:
however you prefer. I thought the PR had to go in first as tensorboard is github first?
but for me, updating google side is the important part.

Reply:
Just wanted to check, the last build failures seem to be due to download errors.
Would you like me to inspect the build failures here, or the build logs on the internal cl?

Reply:
In our process we developed in GH first. I just wonder if it's more convenient to you to have an internal cl (I guess you are using internal script or so to add those checks?)
So I would say which you prefer and/or more convenient for you to update those type syntax then.
If you're going to examine this in OSS and willing to build the env (steps provided in b/), you can also patch this PR and add fixes. However you prefer

Regards to the error, I'm not sure what you mean by download errors. They seem like type check to me 
<img width="1018" alt="Screen Shot 2022-09-29 at 9 26 21 AM" src="https://user-images.githubusercontent.com/1131010/193086708-d75dbf66-1b58-4cf5-baae-86306cf1ddc5.png">

Reply:
I have been confused by these errors:

```
Suppressed: com.google.devtools.build.lib.remote.http.DownloadTimeoutException: Download of '/tensorboard-build-cache/cas/15f51cafb9422a55951f104d456d2854ebfe0bbf4b2bfe9c63e395d14657866a' timed out. Received 0 bytes.
		at com.google.devtools.build.lib.remote.http.HttpDownloadHandler.exceptionCaught(HttpDownloadHandler.java:156)
		at io.netty.channel.AbstractChannelHandlerContext.invokeExceptionCaught(AbstractChannelHandlerContext.java:302)
		at io.netty.channel.AbstractChannelHandlerContext.invokeExceptionCaught(AbstractChannelHandlerContext.java:281)
```

but now I see the errors.

I will setup the local development and patch your branch and build. I will comment the fixes in a bit, hopefully

Reply:
Ah I see. That's an unimportant warning and you can ignore those. The critical part is shown in the screenshot. If that builds the DownloadTimeoutException error might go away. Please ping me if you need anything